### ISSUE TITLE
Firefox accessibility issue no focus on calendar picker can't select a day via keyboard

### ISSUE THREAD

**Issue by MatteoNY** [airbnb/react-dates#717]
Keyboard works well with Chrome and IE Edge. The calendar picker opens but the down-arrow or tab don't focus on a date to select. Works fine if you click. Not using autofocus but that feature only opens the calendar and I need to have multiple instances of the calendar in some use cases. You can test with "Storybook" it works sometimes but not stable as Chrome and IE Edge.  Thanks in advance for your help.

**Comment by erin-doyle** [comment]
I think this is because the default behavior of the arrow keys in FireFox is to scroll the page up and down.  It seems like after focusing the Date Input a second time it works ok from then.  I'm looking in to it.

**Comment by erin-doyle** [comment]
After more testing:
I can't reproduce this with the Storybook.

I was able to reproduce within my own app that uses react-dates but what I was finding with respect to the arrow-down key not transitioning focus to a date was because the DateInput was not focused when I initially would tab to it.  Something about the way my app was managing the focused prop would cause the DateInput to blur() and then focus().  Even though the DateInput would focus() Firefox doesn't seem to recognize it until the field is focused again.  The difference I could see in Chrome was that for some reason Chrome was automatically focusing on the input can causing DateInput to call focus() a second time.  So in Chrome it ended up working ok.

I have modified the way my app was managing the focused prop to prevent the DateInput calling blur() and then focus().  Now the DateInput receives focus on initial tab to it and I can use the down-arrow the first time to focus into the Calendar.  I do still have a remaining issue where after the DateInput calls onBlur() when I hit tab the focus doesn't move to the next element on my page but comes back to the DateInput again.

Bottom line I _think_ this is FireFox related behavior and not something that is being caused by react-dates.  I could be wrong but I feel like once all of the respective react-dates components being used have blurred the focus is no longer being controlled by react-dates.  The behavior of where focus goes next is in the browser's realm of responsibility in my estimation.

If anyone else has any other ideas or can prove my theory wrong I'd be very happy to hear them.  I've been spending a lot of time debugging this one and it's quite a head scratcher to me at this point.

**Comment by erin-doyle** [comment]
This is pretty old and A LOT has changed in the library since this time.  I'm going to go ahead and close this.
